pkg/netmap/node_info_test.go

```diff
@@ -179,3 +211,41 @@ func TestNodeInfoEncoding(t *testing.T) {
 		require.Equal(t, i, i2)
 	})
 }
+
+func TestNewNodeAttribute(t *testing.T) {
+	t.Run("default values", func(t *testing.T) {
+		attr := NewNodeAttribute()
+
+		// check initial values
+		require.Empty(t, attr.Key())
+		require.Empty(t, attr.Value())
+		require.Nil(t, attr.ParentKeys())
+
+		// convert to v2 message
+		attrV2 := attr.ToV2()
+
+		require.Empty(t, attrV2.GetKey())
+		require.Empty(t, attrV2.GetValue())
+		require.Nil(t, attrV2.GetParents())
+	})
+}
+
+func TestNewNodeInfo(t *testing.T) {
+	t.Run("default values", func(t *testing.T) {
+		ni := NewNodeInfo()
+
+		// check initial values
+		require.Empty(t, ni.PublicKey())
```

Comment:
```suggestion
		require.Nil(t, ni.PublicKey())
```
Empty slice is empty too